boto3/iam/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Should be we using the current year for new license headers?

Reply:
Yeah i should use current year.